eng/pipelines/templates/stages/cosmos-emulator-matrix.json

```diff
@@ -50,6 +50,13 @@
         "DESIRED_CONSISTENCIES": "[\"Session\"]",
         "JavaTestVersion": "1.8",
         "AdditionalArgs": "-DargLine=\"-DACCOUNT_HOST=https://localhost:8081/ -Dhadoop.home.dir=D:/Hadoop\""
+      },
+      "Spark 3.3 Integration Tests targeting Cosmos Emulator'": {
+        "ProfileFlag": "-Dspark-e2e_3-3",
+        "PROTOCOLS": "[\"Tcp\"]",
+        "DESIRED_CONSISTENCIES": "[\"Session\"]",
+        "JavaTestVersion": "1.8",
```

Comment:
FYI, I thought for spark 3.*, we will need Java > 11. 
@xinlian12 worth checking on this before we merge this PR.

Reply:
fixed.